### PR TITLE
PR: Rename `master` to `main` #8

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] #8 
+ [x] #10

### CODE REVIEW

**Comment by nelsonic** [comment]
```sh
Error 404: javascript-projects
Error 404: elixir-auth-facebook-v1
  63 branches successfully protected
  309 branches errored
```

So basically _most_ of our repos currently have a `master` branch instead of `main` ... 
We need to write _another_ script to rename all the repos with `master` branches to `main` ...

File: lib/protect.ex [hunk 1]
```diff
@@ -56,71 +61,95 @@ defmodule Protect do
...
+  @doc """
+    `rename_master_to_main/2` renames the given repo's default branch from master to main.
+    should fail silently.
+    API Doc: https://docs.github.com/en/rest/branches/branches#rename-a-branch
+    Ref: https://openssl.medium.com/renaming-the-default-branch-from-master-to-main-on-github-2f965cdbbc19
+  """
+  def rename_master_to_main(options, repo) do
+    "/repos/#{owner(options)}/#{repo}/branches/master/rename"
+    |> Github.post!(Poison.encode!(%{new_name: "main"}))
+    |> dbg()
```
Comment (SimonLab) [inline lib/protect.ex:118]: Do you want to keep the `debug` in the code?

File: lib/protect.ex [hunk 2]
```diff
@@ -56,71 +61,95 @@ defmodule Protect do
...
+
+  defp owner(options) do
+    cond do
+      options[:org] ->
+        options[:org]
+
+      options[:user] ->
+        options[:user]
     end
   end
```
Comment (SimonLab) [inline lib/protect.ex]: I think we can simplify this function to:
```suggestion
  defp owner(options), do: options[:org] || options[:user]
```

File: lib/protect.ex (hunk 1 above)
Comment (nelsonic) [reply lib/protect.ex:118]: yeah, want to debug while it's running to get some insight into errors so can attempt to fix them. 💭

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

**Comment by nelsonic** [comment]
Merci @SimonLab 🙏

**Pull request merged**